tornado/http1connection.py

```diff
@@ -159,6 +159,10 @@ def read_response(self, delegate):
     @gen.coroutine
     def _read_message(self, delegate):
         need_delegate_close = False
+        if self.is_client:
```

Comment:
Why not `is_client = self.is_client`?

Reply:
This forces the local to be of type `bint` in Cython